libs/features/mas/src/upt-link.js

```diff
@@ -0,0 +1,80 @@
+import { useService as getService } from "./utilities.js";
+
+const PROMO_TERMS_URL = 'https://www.adobe.com/offers/promo-terms.html';
```

Comment:
i wonder if we will be asked to also support 'https://www.stage.adobe.com' when mas-commerce-service env=stage